frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -276,6 +276,8 @@ class BitBox02 extends Component<Props, State> {
                 }, () => {
                     this.setPassword();
                 });
+                // show noPasswordMatch error and do NOT continue to createBackup
+                return;
             }
             this.setState({ settingPassword: false, createWalletStatus: 'createBackup' });
```

Comment:
should this maybe be also after the `set-password` response arrived, in case if `success`?

Reply:
sry i misread and thought this was outside the response callback. all good